config/crd/bases/anywhere.eks.amazonaws.com_tinkerbelldatacenterconfigs.yaml

```diff
@@ -0,0 +1,71 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
sorry to be a pain but can we move these two to a separate PR (I don't mind taking it)?

Reply:
Sureee